released/packages/coq-dblib/coq-dblib.8.6.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "matej.kosik@inria.fr"
```

Comment:
But I didn't create the `desc`/`opam`/`url` for this package. As I stated, they're copied verbatim from the previous [closed PR](https://github.com/coq/opam-coq-archive/pull/134). As per previous discussion on the `coq-aac-tactics` package, I thought it was OK to submit some select packages from that PR if there was interest.

If you still want me to act as maintainer for this particular package, I'd be OK with that I guess (let me know and I'll make a commit).